website/docs/docs/build/about-metricflow.md

```diff
@@ -17,7 +17,7 @@ Before you start, consider the following guidelines:
 - Define metrics in YAML and query them using these [new metric specifications](https://github.com/dbt-labs/dbt-core/discussions/7456).
 - You must be on [dbt version](/docs/dbt-versions/upgrade-core-in-cloud) 1.6 or higher to use MetricFlow. 
 - Use MetricFlow with Snowflake, BigQuery, Databricks, Postgres (dbt Core only), or Redshift. 
-- Discover insights and query your metrics using the [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) and its diverse range of [available integrations](/docs/use-dbt-semantic-layer/avail-sl-integrations).  You must have a dbt Cloud account on the [Team or Enterprise plan](https://www.getdbt.com/pricing/). 
+- Discover insights and query your metrics using the [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) and its diverse range of [available integrations](/docs/use-dbt-semantic-layer/avail-sl-integrations). 
```

Comment:
It may be a good idea to add a section to the Available Integrations page to indicate how Exports can be used with tools that we don't support, such as PowerBI.